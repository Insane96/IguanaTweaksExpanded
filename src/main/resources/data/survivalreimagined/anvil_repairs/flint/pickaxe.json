{
	"item_to_repair": {
		"id": "survivalreimagined:flint_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 3
		}
	]
}