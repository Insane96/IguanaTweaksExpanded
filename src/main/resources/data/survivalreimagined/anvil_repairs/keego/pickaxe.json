{
	"item_to_repair": {
		"id": "survivalreimagined:keego_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 3,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:keego"
			},
			"amount": 2
		}
	]
}