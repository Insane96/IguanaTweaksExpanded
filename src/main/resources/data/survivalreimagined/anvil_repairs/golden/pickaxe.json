{
	"item_to_repair": {
		"id": "minecraft:golden_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 3
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 27
		}
	]
}